src/components/ProblemPreview/ProblemPreview.tsx

```diff
@@ -0,0 +1,34 @@
+import React, { FC } from 'react';
+import { Text } from '@taskany/bricks/harmony';
+
+import { Preview } from '../Preview/Preview';
+import { ProblemFilterBar } from '../ProblemFilterBar/ProblemFilterBar';
+import { ProblemList } from '../ProblemList/ProblemList';
+
+import { tr } from './ProblemPreview.i18n';
+import s from './ProblemPreview.module.css';
+
+interface ProblemPreviewProps {
+    problemPreview?: {
+        interviewId?: number;
+        sectionId?: number;
+    };
+}
+
+export const ProblemPreview: FC<ProblemPreviewProps> = ({ problemPreview }: ProblemPreviewProps) => {
```

Comment:
This is not a preview of a problem. Why this component is separate from AddProblemToSection?